public_html/index.html

```diff
@@ -27,10 +27,10 @@
 		<tool-navbar>
 			<template slot='left'>
 				<li class="nav-item navthing">
-					<a class='btn btn-outline-secondary' target='_blank' href='https://github.com/magnusmanske/cradle' tt='git'></a>
+					<a class='btn btn-outline-secondary' target='_blank' href='https://github.com/wbstack/cradle' tt='git'></a>
 				</li>
 				<li class="nav-item navthing">
-					<a id='help_page' class='btn btn-outline-info' target='_blank' href='#' tt='help'></a>
+					<a id='help_page' class='btn btn-outline-info' target='_blank' href='https://www.mediawiki.org/wiki/Wikibase/Creating_and_deleting_data/en#Cradle' tt='help'></a>
```

Comment:
I was trying to work out why we had to hard code it rather than push the config to the right place but now I see that it's wanting to go to the `source_page` and we aren't going to install instructions onto each wiki